app/js/main.js

```diff
@@ -658,7 +675,31 @@ function _getDefaultDirectory() {
 }
 
 /**
-* COnverting frames to png
+ * Check the save directory still exists and recreate it if deleted.
+ */
+function checkDirectoryExists() {
```

Comment:
I almost want to rename this to `createSaveDirectory()` (generic name because we will reuse it later in projects).